clients/cobol-lsp-vscode-extension/snippets.json

```diff
@@ -0,0 +1,1938 @@
+{
+	"add": {
+		"prefix": "add",
+		"body": [
+			"add ${1:a} to ${2:b} giving ${3:c}"
+		],
+		"description": "Add a to b giving c",
+		"scope": "cobol,acucobol,opencobol,gnucobol"
+	},
+	"ADD": {
+		"prefix": "ADD",
+		"body": [
+			"ADD ${1:a} TO ${2:b} GIVING ${3:c}"
+		],
+		"description": "Add a to b giving c",
+		"scope": "cobol,acucobol,opencobol,gnucobol"
+	},
```

Comment:
Do we really need them in different cases?

Reply:
> Do we really need them in different cases?

Yes, it is agreed by Scott. Just some dev. may prefer upper case, some lower cases.